plugins/generator-datapack-1.16.x/datapack-1.16.x/templates/dimension/dimension_type.json.ftl

```diff
@@ -11,7 +11,7 @@
   "ambient_light": <#if data.isDark>0<#else>0.5</#if>,
   "logical_height": ${data.logicalHeight},
   "infiniburn": "minecraft:infiniburn_overworld",
-  "min_y": 0, <#-- Minecraft does not support custom heights in this version -->
-  "height": 256, <#-- Minecraft does not support custom heights in this version -->

```

Comment:
I'm sorry you're feeling bad like me today (though I'm not that angry to remove this comma because of selecting the wrong range). Just don't worry, calm down, it happens.